packages/eui/src/components/datagrid/body/header/data_grid_header_row.styles.ts

```diff
@@ -25,15 +25,41 @@ export const euiDataGridHeaderStyles = (euiThemeContext: UseEuiTheme) => {
       ${logicalCSS('width', 'fit-content')}
       display: flex;
     `,
+    euiDataGridHeaderDroppable: css`
```

Comment:
Let's go ahead and make this a `__` element, since it's a child of the header row element
```suggestion
    euiDataGridHeader__droppable: css`
```

Reply:
Also, let's move this down to the bottom of the file if no objections, which is usually where I order child element styles (after modifiers)